Cargo.toml

```diff
@@ -4,4 +4,4 @@ members = ["borsh", "borsh-derive", "fuzz/fuzz-run", "benchmarks"]
 [workspace.package]
 # shared version of all public crates in the workspace
 version = "1.5.1"
-rust-version = "1.67.0"
+rust-version = "1.81.0"
```

Comment:
@dzmitry-lahoda why raising minimal version to this value was needed?

Reply:
No worries. I will remove that, no need it to merge.  

I need suggestion for https://github.com/near/borsh-rs/pull/315#issuecomment-2411930420   .

Will clean up things after.